test/cypress/integration/settings/allow_apple_pay_spec.js

```diff
@@ -2,46 +2,86 @@ describe('Apple Pay', () => {
   const userExternalId = 'cd0fa54cf3b7408a80ae2f1b93e7c16e'
   const gatewayAccountId = 42
   const serviceName = 'My Awesome Service'
+  const paymentProvider = 'worldpay'
 
   beforeEach(() => {
-    cy.setEncryptedCookies(userExternalId, gatewayAccountId)
+    Cypress.Cookies.preserveOnce('session', 'gateway_account')
+  })
 
-    cy.task('setupStubs', [
-      {
-        name: 'getUserSuccess',
-        opts: {
-          external_id: userExternalId,
-          service_roles: [{
-            service: {
-              gateway_account_ids: [gatewayAccountId],
-              name: serviceName
-            }
+  describe('is disabled', () => {
+    beforeEach(() => {
+      cy.task('setupStubs', [
+        {
+          name: 'getUserSuccess',
+          opts: {
+            external_id: userExternalId,
+            service_roles: [{
+              service: {
+                gateway_account_ids: [gatewayAccountId],
+                name: serviceName
+              }
+            }]
+          }
+        },
+        {
+          name: 'getGatewayAccountSuccessRepeatNTimes',
```

Comment:
I don't think you need to use this repeated stub, this is only intended for when the same request will return a different response on subsequent calls, whereas the response is the same both times here.

Reply:
Yeah i didnt know what that did but was just there in the test already 🤷‍♂, will refactor 👍